src/modules/auth/application/use-cases/renew-token.use-case.ts

```
@@ -30,14 +30,16 @@ export class RenewTokenUseCase {
   execute(refreshToken: string) {
     try {
       const payload = this.jwtService.verify<
-        JwtPayload & { isTwoFactorEnabled?: boolean; role?: any }
+        JwtPayload & { isTwoFactorEnabled?: boolean; role?: any; roles?: any[]; isActive?: boolean }
```

ESLint

[error] Replace `·isTwoFactorEnabled?:·boolean;·role?:·any;·roles?:·any[];·isActive?:·boolean` with `⏎··········isTwoFactorEnabled?:·boolean;⏎··········role?:·any;⏎··········roles?:·any[];⏎··········isActive?:·boolean;⏎·······`

(prettier/prettier)